components/MCQ/mcq.tsx

```diff
@@ -1,37 +1,46 @@
 import { RadioGroup, Radio } from "@mantine/core";
 import React, { useState } from "react";
 import { Props } from "./mcq.types";
-import { Image } from '@mantine/core';
+import { Image } from "@mantine/core";
 import { imageStyles } from "./styles";
+import { Grid } from "@mantine/core";
 export default function MCQ(props: Props) {
-  const { question, description, required, options, orientation,image,alt } = props;
+  const { question, description, required, options, orientation, image, alt } = props;
   const [value, setValue] = useState<string>("1");
+  let span=orientation=="horizontal"?8:3;
   return (
-    <div style={{display: 'flex'}}>
-    <RadioGroup
-      label={question}
-      orientation={orientation}
-      description={description}
-      required={required}
-      value={value.toString()}
-      onChange={(value)=> setValue(value)}
-    >
-      
-      {options.map((option) => (
-        <Radio key={option.value} value={option.value.toString()} label={option.labelText}></Radio>
-      ))}
-    </RadioGroup>
-    {image ? (<Image sx={imageStyles} radius="md" radioGroup="Question"  src={image} alt={alt?alt:"Image about Question"}/>):<></>} 
-    </div>
+    <Grid>
+      <Grid.Col span={span}>
+        <RadioGroup
+          label={question}
+          orientation={orientation}
+          description={description}
+          required={required}
+          value={value.toString()}
+          onChange={(value) => setValue(value)}
+        >
+          {options.map((option) => (
+            <Radio
+              key={option.value}
+              value={option.value.toString()}
+              label={option.labelText}
+            ></Radio>
+          ))}
+        </RadioGroup>
+      </Grid.Col>
+      <Grid.Col span={3}>
```

Comment:
This will always create a `Col` regardless image exists or not. resulting in parent grid always devided in 2 parts. what you should do is create this col only if Grid exists

Reply:
eg
```jsx
<Grid>
<Grid.Col></Grid.Col>
{image ? <Grid.Col></Grid.Col> : null}
</Grid>
```

Reply:
Ok, sure I will do that. It was a really bad mistake from my end.